app/views/kss/home/styleguide2.html.erb

```diff
@@ -0,0 +1,742 @@
+          <button type="button" class="close" data-dismiss="alert" aria-hidden="true">×</button>
+          WAT!!
+        </div>
+
+        <div id="application-error" class="alert alert-info alert-dismissable application-error" style="position: inherit;">
+          <button type="button" class="close" data-dismiss="alert" aria-hidden="true">×</button>
+          Hey, Check this out!
+        </div>
+
+        <div id="application-error" class="alert alert-warning alert-dismissable application-error" style="position: inherit;">
+          <button type="button" class="close" data-dismiss="alert" aria-hidden="true">×</button>
+          Ugh..!
+        </div>
+
+        <div id="application-error" class="alert alert-success alert-dismissable application-error" style="position: inherit;">
+          <button type="button" class="close" data-dismiss="alert" aria-hidden="true">×</button>
+          YES!!
+        </div>
+
+        <hr>
+
+        <h2 id="tooltips">
+          Tooltips
+        </h2>
+        <div element-name="three" source-page-name="dashboard" source-page-selector=".welcome-message"></div>
+
+        <hr>
+
+        <h2 id="card-things">
+          Card Things
+        </h2>
+
+        <div element-name="three" source-page-name="dashboard" source-page-selector=".welcome-message"></div>
+
+        <hr>
+
+        <h2 id="widgets">
+          Widgets
+        </h2>
+        <div element-name="three" source-page-name="dashboard" source-page-selector=".welcome-message"></div>
+
+        <hr>
+
+        <h2 id="avatars">
+          Avatars
+        </h2>
+
+        <div class="row">
+          <div class="col-md-4">
+            <div id="profile-avatar">
+              <img src="/images/profile-no-image.png" data-bindattr-1612="1612">
+              <div id="profile-avatar-hover">
+                <span class="button-secondary button--green fileinput-button">
+                  <span>Upload New</span>
+                  <input id="profile_avatar" class="ember-text-field" type="file" name="file" accept=".jpg,.jpeg,.tiff,.gif,.png">
+                </span>
+              </div>
+              <div class="profile-avatar-spinner"></div>
+            </div>
+          </div>
+          <div class="col-md-4">
+            <img src="/images/profile-no-image.png" data-bindattr-1041="1041" class="user-thumbnail">
+          </div>
+          <div class="col-md-4">
+            <img alt="User Name" class="user-thumbnail-small" src="/images/profile-no-image.png" data-toggle="tooltip" title="" data-original-title="User Name">
+          </div>
+        </div>
+
+        <hr>
+
+        <h2 id="task-cards">
+          Task Cards
+        </h2>
+        <div element-name="three" source-page-name="dashboard" source-page-selector=".welcome-message"></div>
+
+      </div>
+
+    </div>
+  </div>
+
+  <div class="navigation" style="left: 0; display: none;">
```

Comment:
Incorporate this Tahi navigation into the guide.  Move toward a Bootstrap-like scrollspy for styleguide nav.